src/codeu/chat/util/Serializers.java

```diff
@@ -151,6 +154,68 @@ public void write(OutputStream out, Collection<T> value) throws IOException {
     };
   }
 
+  public static <K, V> Serializer<Map<K, V>> map(final Serializer<K> keySerializer, final Serializer<V> valueSerializer) {
```

Comment:
Convention is to use CAPS for static members in this class (so `MAP` instead of `map`)